docs/advanced_topics/settings.rst

```diff
@@ -394,9 +394,13 @@ Usage for images, documents and snippets
 
     WAGTAIL_USAGE_COUNT_ENABLED = True
 
-When enabled Wagtail shows where a particular image, document or snippet is being used on your site (disabled by default). A link will appear on the edit page showing you which pages they have been used on.
+When enabled Wagtail shows where a particular image, document or snippet is being used on your site.
+This is disabled by default because of performance concerns.
```

Comment:
What are these concerns? We should spell them out.

Reply:
I suggest 'This is disabled by default because it generates a query which may run slowly on sites with large numbers of pages'.